README.md

```diff
@@ -10,20 +10,32 @@ There are three main components in this repository: the circuit layout (file [ci
 
 For building the measurement device, you need the following parts:
 - Arduino Mega 2560: Does not have to be original Arduino, can also be e.g. a SunFounder Mega 2560
-- LED: A Light-emitting diode, e.g. LED 5-4500 RT
-- Phototransistor: For example the SDP 8406-003
-- Resistor 11kOhm: Any 11kOhm resistor will do the job, e.g. the 1/4W 11K
+- LED: A Light-emitting diode, e.g. `LED 5-4500 RT`. You might need to play around with these - depending on the colour and brightness of the LED, the PT may have a harder time detecting the LED on your screen
+- Phototransistor: For example the `SDP 8406-003`
+- Resistor 11k$\Omega$: Any 11k$\Omega$ resistor will do the job, e.g. the 1/4W 11K
+- Resistor 220$\Omega$: A resistor around this value will limit the current draw from the LED and protect the Arduino pin
 - Wires
-- Optional: A breadboard
+- Optional: A breadboard, protoboard or Arduino shield
+- Optional: 3D-printed case with three to seven `M3x10` screws and three to seven `278 534` (or equivalent) threaded inserts
 
-Next, connect the elements as such:
+Next, you have a few options for building the circuit. Either way, make sure that pin A0 is not connected to anything (or pick a new free analog pin for it) - it needs to be floating for the randomness to work properly.
+
+If you make it on an Arduino shield, you can 3D print [this case](https://grabcad.com/library/glass2glass-arduino-mega-case-1) for it and do something like this:
```

Comment:
Nice! Should we give more details to the readers about how to construct this? I'm not sure whether the images are sufficient for e.g. beginners.

Reply:
The case is just two parts, and I provided the full SolidWorks assembly which shows how all of the hardware is put together (no circuit on the protoboard though), so to me that's ok for the hardware. If they have access to a 3D printer they probably won't have trouble with threaded inserts.

The other part would be explaining how to solder the protoboard, and I feel like that should also be prerequisite knowledge if they're going through with using the case. Only other thing I might add would be a comment on the wire gauge/type and length, I picked something thick but it looks like in your paper you used really thin wires

Reply:
Sorry for being unclear: I agree that 3D printing is covered by SolidWorks. I was more concerned with the exact circuit you put together. But now, when looking at your photos of the front and back of the shield, I think that these are absolutely sufficient. In particular when embedded into the explanatory text.

Reply:
okay, thanks for the clarification. I'll respond to the other points when I get the chance!